scanner/Makefile

```diff
@@ -23,11 +23,9 @@ HOST_OS := linux
 ifeq ($(shell uname -s),Darwin)
 	HOST_OS := darwin
 endif
-
-CGO_ENABLED := 0
 GOOS := $(HOST_OS)
```

Comment:
I restored the original `GO_BUILD_FLAGS` because I checked and I liked how it looks in the build output and how it makes things explicit.

![image](https://github.com/stackrox/stackrox/assets/537715/af141c02-99ed-4d2a-8dcb-1dcf54cae282)
versus
![image](https://github.com/stackrox/stackrox/assets/537715/10bfb2f9-1106-401e-b619-0d1e377001dc)